packages/fast-components-react-msft/src/carousel/README.md

```diff
@@ -1,3 +1,7 @@
 # Carousel
 
 *Carousel* allows the user to page through content blocks one at a time or arrow through a sequence of smaller content blocks.
```

Comment:
Would arrow key also work for paging through content blocks?  Is "page" the most accurate way of describing this?

Reply:
This is carousel text from older documentation... I could see this as a bit convoluted though.

"Carousel allows the user to move through a series of slides" could be a better way of saying this.

Reply:
I think that would be an improvement. please change

Reply:
Changed